diff --git a/node_modules/codemirror/mode/mllike/mllike.js b/node_modules/codemirror/mode/mllike/mllike.js
index a21a555..ff6b705 100644
--- a/node_modules/codemirror/mode/mllike/mllike.js
+++ b/node_modules/codemirror/mode/mllike/mllike.js
@@ -171,6 +171,7 @@ CodeMirror.defineMIME('text/x-ocaml', {
     'done': 'keyword',
     'downto': 'keyword',
     'external': 'keyword',
+    'for': 'keyword',
     'function': 'keyword',
     'initializer': 'keyword',
     'lazy': 'keyword',
diff --git a/node_modules/codemirror/mode/python/python.js b/node_modules/codemirror/mode/python/python.js
index d75b021..56ae9b8 100644
--- a/node_modules/codemirror/mode/python/python.js
+++ b/node_modules/codemirror/mode/python/python.js
@@ -20,6 +20,7 @@
                         "def", "del", "elif", "else", "except", "finally",
                         "for", "from", "global", "if", "import",
                         "lambda", "pass", "raise", "return",
+                        "match", "case",
                         "try", "while", "with", "yield", "in"];
   var commonBuiltins = ["abs", "all", "any", "bin", "bool", "bytearray", "callable", "chr",
                         "classmethod", "compile", "complex", "delattr", "dict", "dir", "divmod",
